flake.nix

```diff
@@ -141,12 +136,12 @@
               ./system/formats/netboot-kexec.nix
               ./system/global.nix
               ./system/ramdisk.nix
-              ./home-manager/core.nix
+              ./home-manager/juuso.nix
+              ./home-manager/kari.nix
```

Comment:
We might want to rethink this after all -- it seems like the home-manager module pulls a huge amount of depdencies on the servers, including a GUI. I think there is a case to be made that should the core user only be used and these users removed, alongside the whole HM module.